### ISSUE TITLE
Role Testing: Site Editor Settings permissions

### ISSUE THREAD

**Issue by cathysnider** [CuBoulder/express#2205]
Test site: https://www-dev-https.colorado.edu/p12984fd01fe/
Note: currently there are no test users with the new roles. I have set the OSR-Test-Owner with Site Editor permissions. I.e. Login as a the OSR-Test-Owner to functionally test the Site Editor role. 

Site Editors should not be able to enable bundles. These pages should return 'Access Denied'
https://www-dev-https.colorado.edu/p12984fd01fe/admin/settings/bundles/list 

Site Editors should not be able to clear Varnish Cache or Database Cache. These pages should return 'Access Denied'
https://www-dev-https.colorado.edu/p12984fd01fe/admin/settings/cache/clear/varnish-full
https://www-dev-https.colorado.edu/p12984fd01fe/admin/settings/cache/clear/drupal-full

Site Editors should not be able to set the site name or contact info. These pages should return 'Access Denied'
https://www-dev-https.colorado.edu/p12984fd01fe/admin/settings/site-configuration/site-name
https://www-dev-https.colorado.edu/p12984fd01fe/admin/settings/site-configuration/contact

Site Editors should not be able to Enable Bundles. These pages should return 'Access Denied'
https://www-dev-https.colorado.edu/p12984fd01fe/admin/settings/bundles/list
https://www-dev-https.colorado.edu/p12984fd01fe/admin/settings/bundles/list/addon
https://www-dev-https.colorado.edu/p12984fd01fe/admin/settings/bundles/list/request

Site Editors should not be able to Set Error Pages. These pages should return 'Access Denied'
https://www-dev-https.colorado.edu/p12984fd01fe/admin/settings/adv-content/error

Site Editors should not be able to access the Feedback Form Settings. Should get Access Denied
https://www-dev-https.colorado.edu/p12984fd01fe/admin/settings/forms/feedback

okay, this list is getting way too long.
Basically, Site Editors give the same permissions as Content Editors on everything, with the following differences:
 - Additions: SEs have Digital Campaign bundle permissions.
 - Exceptions: SEs can't access Menus ; Page Layout or Context. Cannot delete nodes, beans or blocks.
